src/Action/CreditMemoDetailsAction.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Action;
+
+use Doctrine\Common\Persistence\ObjectRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+final class CreditMemoDetailsAction
+{
+    /** @var ObjectRepository */
+    private $creditMemoRepository;
+
+    /** @var Environment */
+    private $twig;
+
+    public function __construct(ObjectRepository $creditMemoRepository, Environment $twig)
+    {
+        $this->creditMemoRepository = $creditMemoRepository;
+        $this->twig = $twig;
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $creditMemo = $this->creditMemoRepository->findOneBy([
+            'orderNumber' => $request->attributes->get('orderNumber'),
+            'id' => $request->attributes->get('id'),
+        ]);
+
+        return new Response(
+            $this->twig->render('@SyliusRefundPlugin/Order/Admin/CreditMemo/details.html.twig', [
+                'creditMemo' => $creditMemo,
```

Comment:
`creditMemo` => `credit_memo`, because we had a convention to use snake case in twig, but I'm not sure if we are still using that.

Reply:
I have a dissonance when I have `credit_memo` variable in twig, but I'm getting the field with camelCase `{{ credit_memo.currencyCode }}` :(

Reply:
It's ok for now, but I would be for thinking about some convention in the nearest future.